source/contracts/reporting/Universe.sol

```diff
@@ -118,7 +118,7 @@ contract Universe is DelegationTarget, Extractable, ITyped, Initializable, IUniv
     }
 
     function getFeeWindowId(uint256 _timestamp) public view returns (uint256) {
-        return _timestamp / getDisputeRoundDurationInSeconds();
+        return _timestamp.div(getDisputeRoundDurationInSeconds());
```

Comment:
Using these functions is more expensive than the native math.  In a lot of these cases, we can easily assert that the math is safe with a comment (which we do in a couple places elsewhere).  While none of these changes are incorrect, I do think we should make sure we are asking if they are worth the increased gas costs or not.

Reply:
The cases where I added safe math checks involve user input so i believe it is appropriate to do this as a validation measure.

Reply:
I have seen that strategy followed before (license file has multiple licenses and what they apply to), which is why I mentioned it.  The most common strategy (by a large margin) is to do what we did before and simply not give attribution when something is licensed as "open source".  😄

My general argument is simply that we should _not_ be littering code with license files.  How we achieve that is less important to me, though I do like encapsulating all of the licensing theater into a single file in the root of the project that everyone can ignore.